kinto/tests/test_config.py

```diff
@@ -0,0 +1,15 @@
+from kinto.config import render_template, HERE, init
+import os
+import unittest
+
+class ConfigTest(unittest.TestCase):
+
+#to check if values dict is not empty    
+        
+#To check if rendered template generates a file
+   
+    def test_init(self):
+        
+        destination = os.path.join(os.path.abspath(os.path.dirname(__file__)), test.ini)
```

Comment:
You probably want to use a temporary file. [Using mktemp](https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp)